docs/PokemonCreation.md

```diff
@@ -0,0 +1,68 @@
+## Pokemon Creation
+
+Pokemon creation in FullScreenPokemon consists of three different types, Unit Test Pokemon creation, Trainer Battle Pokemon creation, and Wild Pokemon Grass Encounter creation.
```

Comment:
It's improper to capitalize the phrases like "Unit Test Pokemon" inline unless they're linked titles.